src/pages/home/sidebar/OptionRowTitle/index.js

```diff
@@ -0,0 +1,124 @@
+/* eslint-disable react/forbid-prop-types */
+import _ from 'underscore';
+import React, {Fragment, PureComponent} from 'react';
+import {
+    Text,
+    View,
+} from 'react-native';
+import propTypes from './OptionRowTitleProps';
+import styles from '../../../../styles/styles';
+import Tooltip from '../../../../components/Tooltip';
+import hasEllipsis from '../../../../libs/hasEllipsis';
+
+const defaultProps = {
+    style: null,
+    tooltipEnabled: false,
+};
+class OptionRowTitle extends PureComponent {
+    constructor(props) {
+        super(props);
+        this.ref = React.createRef();
+        this.cRefs = [];
+        this.state = {
+            isEllipsisActive: false,
+            containerLayout: null,
+        };
+        this.setContainerLayout = this.setContainerLayout.bind(this);
+        this.getTooltipShiftX = this.getTooltipShiftX.bind(this);
+    }
+
+    componentDidMount() {
+        this.setState({
+            isEllipsisActive: this.ref.current && hasEllipsis(this.ref.current),
+        });
+        this.cRefs = this.props.option.participantsList.map(() => React.createRef());
```

Comment:
Check out the [section of our style guide on refs](https://github.com/Expensify/Expensify.cash/blob/master/STYLE.md#use-refs-appropriately). For the sake of consistency, we do not use `createRef` anywhere in Expensify.cash. So you'll need to refactor this component to use callback refs instead.